arm-storage/2016-05-01/swagger/storage.json

```diff
@@ -0,0 +1,1327 @@
+      "properties": {
+        "signedServices": {
+          "type": "string",
+          "enum": [
+            "b",
+            "q",
+            "t",
+            "f"
+          ],
+          "x-ms-client-name": "Services",
+          "description": "Sets the signed services accessible with the account SAS."
+        },
+        "signedResourceTypes": {
+          "type": "string",
+          "enum": [
+            "s",
+            "c",
+            "o"
+          ],
+          "x-ms-client-name": "ResourceTypes",
+          "description": "Sets the signed resource types that are accessible with the account SAS."
+        },
+        "signedPermission": {
+          "type": "string",
+          "enum": [
+            "r",
+            "d",
+            "w",
+            "l",
+            "a",
+            "c",
+            "u",
+            "p"
+          ],
+          "x-ms-client-name": "Permissions",
+          "description": "Sets the signed permissions for the account SAS."
+        },
+        "signedIp": {
+          "type": "string",
+          "x-ms-client-name": "IPAddressOrRange",
+          "description": "Sets an IP address or a range of IP addresses from which to accept requests."
+        },
+        "signedProtocol": {
+          "type": "string",
+          "enum": [
+            "https,http",
+            "https"
+          ],
+          "x-ms-enum": {
+            "name": "HttpProtocol",
+            "modelAsString": false
+          },
+          "x-ms-client-name": "Protocols",
+          "description": "Sets the protocol permitted for a request made with the account SAS."
+        },
+        "signedStart": {
+          "type": "string",
+          "format": "date-time",
+          "x-ms-client-name": "SharedAccessStartTime",
+          "description": "Sets the time at which the SAS becomes valid."
+        },
+        "signedExpiry": {
+          "type": "string",
+          "format": "date-time",
+          "x-ms-client-name": "SharedAccessExpiryTime",
+          "description": "Sets the time at which the shared access signature becomes invalid."
+        },
+        "keyToSign": {
+          "type": "string",
+          "description": "Sets the key to sign the account SAS token with."
+        }
+      },
+      "required": [
+        "signedServices",
+        "signedResourceTypes",
+        "signedPermission",
+        "signedExpiry"
+      ],
+      "description": "The parameters to list SAS credentials of a storage account."
+    },
+    "ListAccountSasResponse": {
+      "properties": {
+        "accountSasToken": {
+          "readOnly": true,
+          "type": "string",
+          "description": "List SAS credentials of storage account."
+        }
+      },
+      "description": "The List SAS credentials operation response."
+    },
+    "ServiceSasParameters": {
+      "properties": {
+        "canonicalizedResource": {
+          "type": "string",
+          "x-ms-client-name": "CanonicalizedResource",
```

Comment:
How is the client-name different from the property name except it being PascalCase. This property will be pascal cased any ways in C# as the model property name and the serialized name would remain as is.  In javascript no matter what the model property name would remain camelCase. So, I don't see the need over here. Please remove this extension.